components/docs-chef-io/content/automate/High_Availability.md

```diff
@@ -0,0 +1,307 @@
++++
+title = "High Availability"
+
+draft = false
```

Comment:
```suggestion
draft = true
```

This way, we can merge them without publication

Reply:
ok